diff -urN src/src/gallium/auxiliary/util/u_debug_symbol.c src_fix/src/gallium/auxiliary/util/u_debug_symbol.c
--- src/src/gallium/auxiliary/util/u_debug_symbol.c	2025-02-28 18:05:34.037328395 +0800
+++ src_fix/src/gallium/auxiliary/util/u_debug_symbol.c	2025-02-28 18:03:02.533035302 +0800
@@ -270,7 +270,7 @@
    debug_printf("\t%s\n", buf);
 }
 
-struct util_hash_table* symbols_hash;
+struct util_hash_table* symbols_hash_symbol;
 static mtx_t symbols_mutex = _MTX_INITIALIZER_NP;
 
 static unsigned hash_ptr(void* p)
@@ -302,16 +302,16 @@
 #endif
 
    mtx_lock(&symbols_mutex);
-   if(!symbols_hash)
-      symbols_hash = util_hash_table_create(hash_ptr, compare_ptr);
-   name = util_hash_table_get(symbols_hash, (void*)addr);
+   if(!symbols_hash_symbol)
+   symbols_hash_symbol = util_hash_table_create(hash_ptr, compare_ptr);
+   name = util_hash_table_get(symbols_hash_symbol, (void*)addr);
    if(!name)
    {
       char buf[1024];
       debug_symbol_name(addr, buf, sizeof(buf));
       name = strdup(buf);
 
-      util_hash_table_set(symbols_hash, (void*)addr, (void*)name);
+      util_hash_table_set(symbols_hash_symbol, (void*)addr, (void*)name);
    }
    mtx_unlock(&symbols_mutex);
    return name;
